src/Bridges/Constraint/bridges/indicator_flipsign.jl

```diff
@@ -0,0 +1,185 @@
+# Copyright (c) 2017: Miles Lubin and contributors
+# Copyright (c) 2017: Google Inc.
+#
+# Use of this source code is governed by an MIT-style license that can be found
+# in the LICENSE.md file or at https://opensource.org/licenses/MIT.
+
+"""
+    IndicatorSetMapBridge{T,A,S1,S2} <: Bridges.Constraint.AbstractBridge
+
+`IndicatorSetMapBridgeBridge` implements the following reformulations:
+
+  * ``z \\implies {f(x) \\ge l}`` into ``z \\implies {-f(x) \\le -l}``
+  * ``z \\implies {f(x) \\le u}`` into ``z \\implies {-f(x) \\ge -u}``
+
+## Source node
+
+`IndicatorSetMapBridgeBridge` supports:
+
+  * [`MOI.VectorAffineFunction{T}`](@ref) in [`MOI.Indicator{A,S1}`](@ref)
+
+## Target nodes
+
+`IndicatorSetMapBridgeBridge` creates:
+
+  * [`MOI.VectorAffineFunction{T}`](@ref) in [`MOI.Indicator{A,S2}`](@ref)
+"""
+struct IndicatorSetMapBridgeBridge{T,B,S1,S2,A} <: AbstractBridge
```

Comment:
```suggestion
struct IndicatorSetMapBridge{T,B,S1,S2,A} <: AbstractBridge
```

Reply:
Hmm. I thought I pushed a commit fixing this. Looks like I didn't.